electives/80405.md

```diff
@@ -0,0 +1,15 @@
+
+# 80-405: Game Theory
+---
+| Category | Difficulty |
+| -------- | ---------- |
+| Homeworks |   3     |
+| Exams |   None   |
+
+Game theory is a philosophy department course that served as an introductory to basic game theories. The main topics include Minimax Theory, Nash Equilibrium, Sequential Games, Bayesian Games, and Bargaining Theory. Scenarios include zero-sum games, auctions, votes, prisoner's dilemma, and sequential games. Although the class informally assumes some knowledge about decision theory, it is an introductory class, and all of the decision theory parts are easy to understand, as it is not in-depth. There are no prerequisites for this class, nor is there a requirement for a solid mathematic background.
+## Lectures
+The lectures cover everything about the course material, and attending lectures would be sufficient to complete all homework. Lecture notes are provided beforehand (and encouraged to read before the lecture), but example problems in the lecture notes were not worked out (You still need to attend the lecture to see the worked out problems). At the last 30-60 minutes of the lecture, TA will typically take over and students will form small groups to discuss the model presented in the lecture.
+## Homeworks
+Homeworks are the only element that counts as the grade in this course. Yes, this course does not have exams. Homeworks is typically composed of 3-4 problems that cover the topic of interest, and it is often a variant of the problems we solved in the class. There are about 5 homework assignments throughout the semester, and each homework is equally weighted and spaced throughout the semester. Each homework assignment typically spans a week and a half, and extensions can be granted once the professor is noticed 24 hours before the deadline. The grading of the homework is dependent on the TA grader. At the beginning of the semester, each small student group will have a TA assigned, and that TA will be responsible for all questions and all gradings. From previous course data, graders were relatively lenient.
+## Textbooks
+The textbook of this class is Osborne's **An Introduction to Game Theory**, which you can find in many sources. The lecture was built around the content of this book, and reading this book can be very helpful, especially it is oftentimes assigned reading. If you are interested in game theory, this book is going to be useful even after the class.
```

Comment:
```suggestion
The textbook of this class is Osborne's **An Introduction to Game Theory**, which you can find in many sources. The lecture was built around the content of this book, and reading this book can be very helpful, especially because it is oftentimes assigned reading. If you are interested in game theory, this book is going to be useful even after the class.
```